package.json

```diff
@@ -17,7 +17,9 @@
     "react": "^18.3.1",
     "react-bootstrap": "^2.10.4",
     "react-dom": "^18.3.1",
+    "react-drag-drop-files": "^2.3.10",
```

Comment:
No need for any packages for the file upload. The file input by default is droppable. We can make a div that contain the styles that we want and an input. and make both of them take the full width and height and then make the opacity of the input 0. This will hide the input, but when clicking on the container, we will be able to select files and if we drop a file, it will also get added. We can position the elements using relative and absolute positioning.